### PR TITLE
Implemented the params from, to, canceled,direction and limit.

### PR DESCRIPTION
**Pull request by kolibri52**
This solves Issue #3 and also #2

### CODE REVIEW

File: ChurchToolsApi.py [hunk 1]
```diff
@@ -477,18 +477,35 @@ def get_songs_with_tag(self, song_tag_id: int):
 
         return result
 
-    def get_events(self, event_id=None):
-        """
-        Retrieve list of events from ChurchTools
-        :return: a list of events
-        #TODO 2 There are params available which are not implemented yet!
+    def get_events(self, from_: str = '', to: str = '', canceled: bool = False, direction: str = 'forward',
```
Comment (bensteUEM) [inline ChurchToolsApi.py:480]: style question - i've started to use kwargs instead of defaulting params for optional params - which way do you prefer?

**Review by bensteUEM — APPROVED** [review]

File: ChurchToolsApi.py (hunk 1 above)
Comment (bensteUEM) [reply ChurchToolsApi.py:480]: You didn't happen to write test methods for the new params did you? I would like to add them before merging into master.

**Comment by bensteUEM** [comment]
You didn't happen to write test methods for the new params?
I would prefer to have tests before merging into master. Should i write them for my test system?

File: ChurchToolsApi.py (hunk 1 above)
Comment (kolibri52) [reply ChurchToolsApi.py:480]: OK. kwargs is a good way.
I will also use it.

**Comment by kolibri52** [comment]
> You didn't happen to write test methods for the new params? I would prefer to have tests before merging into master. Should i write them for my test system?

Yes that would be great. If I would upload a testfunction which is set up for my testenvironment / churchtools-installation it would be mixed up for you.
Therefore I've online tested it in my local Main function.

**Comment by bensteUEM** [comment]
I'll also change it to kwargs and add id param...

**Pull request merged**